lnrpc/wtclientrpc/wtclient.go

```diff
@@ -0,0 +1,298 @@
+// +build wtclientrpc
+
+package wtclientrpc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"strconv"
+
+	"github.com/btcsuite/btcd/btcec"
+	"github.com/lightningnetwork/lnd/lncfg"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/watchtower"
+	"github.com/lightningnetwork/lnd/watchtower/wtclient"
+	"google.golang.org/grpc"
+	"gopkg.in/macaroon-bakery.v2/bakery"
+)
+
+const (
+	// subServerName is the name of the sub rpc server. We'll use this name
+	// to register ourselves, and we also require that the main
+	// SubServerConfigDispatcher instance recognizes it as the name of our
+	// RPC service.
+	subServerName = "WatchtowerClientRPC"
+)
+
+var (
+	// macPermissions maps RPC calls to the permissions they require.
+	//
+	// TODO(wilmer): create tower macaroon?
```

Comment:
This would be nice, as otherwise, the same macaroon (atm) that can add/remove towers is also able to do things like send coins on chain. On the other-hand, one could argue that being able to remove towers is possibly move dangerous than sending coins on chain... 

I think we can leave this out for now, and leave the time from now until the first `0.8` release candidate to mull things over.